optaplanner-core/src/main/java/org/optaplanner/core/api/score/stream/bi/BiConstraintStream.java

```diff
@@ -195,21 +195,27 @@
     }
 
     // ************************************************************************
-    // Group by (TODO implement these)
+    // Group by
     // ************************************************************************
 
-    /*
-    <GroupKey_> UniConstraintStream<GroupKey_> groupBy(
-            BiFunction<A, B, GroupKey_> groupKeyMapping);
+    <GroupKey_> UniConstraintStream<GroupKey_> groupBy(BiFunction<A, B, GroupKey_> groupKeyMapping);
```

Comment:
Shall we immediately add the javadoc if we uncomment it?

Reply:
Good point, will add.